spec/spec_helper.rb

```diff
@@ -14,16 +13,17 @@ module SpecHelper
 end
 
 spec_helper_glob = File.expand_path('{support,shared}/*.rb', __dir__)
-Dir.glob(spec_helper_glob).map(&method(:require))
+Dir.glob(spec_helper_glob).sort.each(&method(:require))
 
 RSpec.configure do |config|
+  # Set metadata so smoke tests are run on all cop specs
   config.define_derived_metadata(file_path: %r{/spec/rubocop/cop/}) do |meta|
     meta[:type] = :cop_spec
   end
 
   # Include config shared context for all cop specs
-  config.define_derived_metadata(type: :cop_spec) do |metadata|
-    metadata[:config] = true
+  config.define_derived_metadata(type: :cop_spec) do |meta|
```

Comment:
I had doubts, but it matches [a canonical example](https://relishapp.com/rspec/rspec-core/v/3-9/docs/expectation-framework-integration/aggregating-failures#enable-failure-aggregation-globally-using-%60define-derived-metadata%60) :+1: